css/src/components/divider.css

```diff
@@ -0,0 +1,6 @@
+/**
+ * Divider.
+ */
+.system-themes-list-uninstalled {
```

Comment:
How about we create a generic divider class that could be attached to these elements?

Reply:
Yes, we can. So I will create classes related to normal divider with border color update, divider with top border style and divider with bottom border style. We can use that classes as per element requirement. Is that fine?

Reply:
How about we make it separate element so that we don't have to worry about border-top and border-bottom? 

For example, we want to add divider before and after the paragraph: 
```
<h1>Title</h1>
<div class="divider"></div>
<p>Paragraph</p>
<div class="divider"></div>
```

Reply:
Ok. You mean to say we could use a separate divider element where we need it like before and after with style height and background color. Right?

Reply:
Yes 👍I also think it would be great if we could convert the existing dividers to use this new element.

Reply:
Also, consider that we need to remove/override the default CSS in this case and template modification because we are adding a separate element.